modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
> Is that what you would say is good 'Ruby' practice or not?

I would try to standardize on the rake-compiler convention, which is also the convention I saw in practice in quite a lot of gems. I.e. having the extension installed under lib/<package_name>/ and required with "<package_name>/extensioname"

Given that extensions are not loaded explicitely - but only through some other ruby files - it should be a change that has no user-visible effect.